### ISSUE TITLE
Update Team Page

### ISSUE THREAD

**Issue by aaronstephenson** [USGS-WiM/WIM-Main-Site#139]
These are probably just quibbles, but...

I noticed that Abby is listed as a "Student Developer" but the other students say, e.g, "Student Software Developer", so maybe change her label to match the others, for consistency?

I also noticed that Kenzie and Nathan are in the team photo at the bottom but not listed in the Alumni section. Or perhaps we just don't list students there?

**Comment by HansVraga** [comment]
Yeah I hadn't planned to add students who move on to "Alumni" - but we could revisit that.

**Comment by HansVraga** [comment]
@mitchas update Abby's title & this one can be closed.
